pxr/usd/usdGeom/schema.usda

```diff
@@ -2353,16 +2382,38 @@ class Camera "Camera" (
         doc = """Frame relative shutter close time, analogous comments from
                  shutter:open apply. A value greater or equal to shutter:open
                  should be authored, otherwise there is no exposure and a
-                 renderer should produce a black image."""
+                 renderer should produce a black image. Used for motion blur."""
     )
     
     # exposure adjustment
     float exposure = 0.0 (
-        doc = """Exposure adjustment, as a log base-2 value.  The default
+        doc = """Exposure compensation, as a log base-2 value.  The default
                  of 0.0 has no effect.  A value of 1.0 will double the
                  image-plane intensities in a rendered image; a value of
                  -1.0 will halve them."""
     )
+
+    # exposure controls
+    float exposure:iso = 100.0 (
+        doc = """The speed rating of the sensor or film when calculating exposure.
+                 Higher numbers give a brighter image, lower numbers darker."""
+    )
+
+    float exposure:time = 1.0 (
```

Comment:
How does this relate to the shutter interval? I assume that in a physical camera, the exposure time is the length of the shutter interval (modulo units of one using frames, the other seconds). I think the comment should say something about it.

Also, the default of 1.0s seems unusually high. I would say 1/60 is more typical. Though, you might subscribe to the philosophy that it is safer to have a non-sensical default value and forcing the user to author the actual value.

Reply:
The default values are specifically constructed such that the exposure scale is exactly 1, thus existing content will continue to work exactly as before. Or to look at it another way, the defaults are the values "currently being used" by a renderer that doesn't yet consider any of this.

The shutter interval is a measure of exposure time, yes, just in different units. Added a clarifying note.

Reply:
Yet another way to look at it is that the default values are what I call "passthrough" values: the output signal (pixel value) is the nits arriving at the sensor.